### PR TITLE
EUI-5101 Hearing stage and result unhappy path

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-5095

### Change description ###
Hearings actuals-Enter hearing stage and result (Happy path)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/hearings/containers/hearing-actuals/hearing-stage-result/hearing-stage-result.component.html [hunk 1]
```diff
@@ -1,82 +1,93 @@
...
+                <input class="govuk-radios__input" id="adjourned" formName="hearingResult" name="hearingResult"
+                  type="radio" value="adjourned" data-aria-controls="conditional-adjourned" (change)="onHearingResult(hearingResultEnum.ADJOURNED)">
+                <label class="govuk-label govuk-radios__label" for="adjourned">
+                  Part-heard, adjourned or postponed
+                </label>
+              </div>
+              <div class="govuk-radios__conditional" *ngIf="hearingResultType === hearingResultEnum.ADJOURNED">
+                <div class="govuk-form-group"
+                  [ngClass]="{'govuk-form-group--error': submitted && adjournHearingErrorMessage.length > 0}">
+                  <label class="govuk-label" id="dropdown-label" for="adjourned-reason">Select a reason for this result</label>
```
Comment (rayliangatos) [inline src/hearings/containers/hearing-actuals/hearing-stage-result/hearing-stage-result.component.html:51]: duplicated 'dropdown-label'

File: src/hearings/containers/hearing-actuals/hearing-stage-result/hearing-stage-result.component.html [hunk 2]
```diff
@@ -1,82 +1,93 @@
...
-                </div>
-                <div class="govuk-radios__conditional" *ngIf="hearingResultType === hearingResultEnum.ADJOURNED">
-                  <div class="govuk-form-group">
-                    <label class="govuk-label" id="dropdown-label" for="adjourned-reason">Select a reason for this result</label>
-                    <select formControlName="adjournedReason" class="govuk-select" id="adjourned-reason" name="adjournedReason">
-                      <option value="" selected>Please select</option>
-                      <option *ngFor="let reason of adjournHearingActualReasons" [ngValue]="reason.key">{{reason.value_en}}</option>
-                    </select>
+        <div class="govuk-form-group"
+          [ngClass]="{'govuk-form-group--error': submitted && !f.valid && !f.hearingResult.valid}">
```
[the remark this answers is not in the document]
Comment (johnbenjamin-hmcts) [reply src/hearings/containers/hearing-actuals/hearing-stage-result/hearing-stage-result.component.html:23]: I have now changed this to formControls.

File: src/hearings/containers/hearing-actuals/hearing-stage-result/hearing-stage-result.component.html (hunk 1 above)
Comment (johnbenjamin-hmcts) [reply src/hearings/containers/hearing-actuals/hearing-stage-result/hearing-stage-result.component.html:51]: Thank you for finding this, I have now changed from id to class.

File: src/hearings/containers/hearing-actuals/hearing-stage-result/hearing-stage-result.component.scss [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#dropdown-label {
+.dropdown-label {
```
Comment (kuda1992) [inline src/hearings/containers/hearing-actuals/hearing-stage-result/hearing-stage-result.component.scss:1]: I think its preferable to use the gov-uk styles `govuk-!-font-weight-bold`

**Review by kuda1992 — CHANGES_REQUESTED** [review]
One minor change

File: src/hearings/containers/hearing-actuals/hearing-stage-result/hearing-stage-result.component.scss (hunk 3 above)
Comment (kuda1992) [reply src/hearings/containers/hearing-actuals/hearing-stage-result/hearing-stage-result.component.scss:1]: https://design-system.service.gov.uk/styles/typography/#font fyi

Comment (johnbenjamin-hmcts) [reply src/hearings/containers/hearing-actuals/hearing-stage-result/hearing-stage-result.component.scss]: Thank you @kuda1992 I have now made the changes.

**Review by rayliangatos — APPROVED** [review]

**Review by kuda1992 — APPROVED** [review]
Looks good

**Pull request merged**